lib-extra/src/main/java/com/diffplug/spotless/extra/groovy/GrEclipseFormatterStep.java

```diff
@@ -68,7 +69,7 @@ private static FormatterFunc apply(EclipseBasedStepBuilder.State state) throws E
 		Method method = formatterClazz.getMethod(FORMATTER_METHOD, String.class);
 		return input -> {
 			try {
-				return (String) method.invoke(formatter, input);
+				return LineEnding.toUnix((String) method.invoke(formatter, input));
```

Comment:
I would remove this change.  The system does it anyway, so it doesn't improve performance.

Reply:
Cool! I will fix this then.